integrations/google-mail/syncs/emails.ts

```diff
@@ -15,8 +15,9 @@ export default async function fetchData(nango: NangoSync) {
     let nextPageToken: string | undefined = '';
 
     do {
+        // https://developers.google.com/gmail/api/reference/rest/v1/users.messages/list
         const response: any = await nango.proxy({
-            method: 'GET',
+            method: 'get',
```

Comment:
should be uppercase no?

Reply:
Both will work; however, when using `GET`, tests will fail because it cannot load the mock data. The mock files are generated under the `get/proxy` folder.